doc/source/ray-core/placement-group.rst

```diff
@@ -487,38 +516,66 @@ because they are scheduled on a placement group with the STRICT_PACK strategy.
 
 .. note::
 
-  In order to fully utilize resources pre-reserved by the placement group,
-  Ray automatically schedules children tasks/actors to the same placement group as its parent.
+  Child actors/tasks don't share the same placement group that the parent uses.
+  If you'd like to automatically schedule child actors/tasks to the same placement group,
+  set ``placement_group_capture_child_tasks`` to True.
 
   .. tabbed:: Python
 
       .. code-block:: python
 
-        # Create a placement group with the STRICT_SPREAD strategy.
-        pg = placement_group([{"CPU": 2}, {"CPU": 2}], strategy="STRICT_SPREAD")
+        import ray
+        import ray
```

Comment:
?

Seems this code snippet is running by itself, can we use `.. literalinclude::` so it's testable.